lib/Dancer2/Core/Role/Config.pm

```diff
@@ -84,9 +91,11 @@ has global_triggers => (
             },
         };
 
+        no warnings 'once'; # Disable: Name "Dancer2::runner" used only once: possible typo
         my $runner_config = defined $Dancer2::runner
                             ? Dancer2->runner->config
                             : {};
+        use warnings 'once';
```

Comment:
Might be easier to write:

```perl
        my $runner_config;
        {
            no warnings 'once';
            $runner_config = defined $Dancer2::runner
                           ? Dancer2->runner->config
                           : {};
        }
```

I would still want to know if we could avoid that warning.

Reply:
Reformatted.